modules/examples/Makefile.am

```diff
@@ -39,4 +40,6 @@ EXTRA_DIST += modules/examples/CMakeLists.txt
 
 modules/examples modules/examples/ mod-examples: modules/examples/libexamples.la
 
+endif
```

Comment:
Shouldn't we have a line like this in an else block?

```
modules/examples modules/examples/ mod-examples:
```

Reply:
sorry, surely my bad, but i do not really get this, this `endif` belongs to the starting `if ENABLE_EXAMPLE_MODULES` that tries to exclude everything in this makefiule if the example modules are disabled

Reply:
We usually have this for our modules (e.g.: https://github.com/syslog-ng/syslog-ng/blob/master/modules/http/Makefile.am#L44)

My understanding was that this is needed so `make mod-examples` does not fail if we configured `--disable-example-modules`, but it seems like it works either way.

@MrAnno do you know why we add empty targets for our modules?

Reply:
They are there to make per-module compilation work.

Reply:
> They are there to make per-module compilation work.

my problem is that it is absolutely stochastic if it's presented or not, and not a strict habit/requirement/rule, so, it's hard to decide if it's really needed to not.